indico/htdocs/js/indico/widgets/person_link_widget.js

```diff
@@ -60,6 +60,12 @@
         var $buttonAlphaOrder = $fieldDisplay.find('.alpha-order-switch');
         var $form = $field.closest('form');
         var customOrder = !$buttonAlphaOrder.hasClass('active');
+        var maxManualPersonID = 0;
+
+        function setManualPersonID(person) {
+            person.id = 'manual-' + maxManualPersonID;
```

Comment:
It seems to be working fine, after saving the form and going back to edit, the manual entered people have received a proper person id